I get some error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.
8.1:testCompile (default-testCompile) on project federation: Compilation failure
: Compilation failure:
[ERROR] /root/cwa-server/common/federation/src/test/java/app/coronawarn/server/c
ommon/federation/client/CloudFederationFeignHttpClientProviderSmokeTest.java:[50
,8] cannot find symbol
[ERROR]   symbol:   method setBaseUrl(java.lang.String)
[ERROR]   location: variable ssl of type app.coronawarn.server.common.federation.client.config.FederationGatewayConfig.Ssl
[ERROR] /root/cwa-server/common/federation/src/test/java/app/coronawarn/server/common/federation/client/CloudFederationFeignHttpClientProviderSmokeTest.java:[51,19] cannot find symbol
[ERROR]   symbol:   method getBaseUrl()
[ERROR]   location: variable ssl of type app.coronawarn.server.common.federation.client.config.FederationGatewayConfig.Ssl
```

--- a/common/federation/src/test/java/app/coronawarn/server/common/federation/client/CloudFederationFeignHttpClientProviderSmokeTest.java
+++ b/common/federation/src/test/java/app/coronawarn/server/common/federation/client/CloudFederationFeignHttpClientProviderSmokeTest.java
@@ -46,9 +46,9 @@ class CloudFederationFeignHttpClientProviderSmokeTest {
 
   @Test
   void testGetBaseUrl(){
-    Ssl ssl = new Ssl();
-    ssl.setBaseUrl("https://github.com/cwa-server/tree/main");
-    assertThat(ssl.getBaseUrl()).isNotNull();
+    FederationGatewayConfig config = new FederationGatewayConfig();
+    config.setBaseUrl("https://github.com/cwa-server/tree/main");
+    assertThat(config.getBaseUrl()).isNotNull();
   }
 
   @Test

